tests/filtertransformers/test_mongo.py

```diff
@@ -1,9 +1,11 @@
 import pytest
 from lark.exceptions import VisitError
 
+from optimade.exceptions import BadRequest
 from optimade.filterparser import LarkParser
-from optimade.server.exceptions import BadRequest
-from optimade.server.warnings import UnknownProviderProperty
+from optimade.warnings import UnknownProviderProperty
+
+pytest.importorskip("bson")
```

Comment:
This statement looks a bit odd. We do not use BSON until much later in the tests and there, there is a separate import statement. So it should not be necessary to import it here.
I assume it is to check whether the mongo deps are present? If so, could you add a comment to explain this?

Reply:
Yeah, this is just the easiest way of skipping an entire pytest module based on whether an import is available. The underlying transformer uses `bson` straight away too. I can add a comment explaining this (and same for elastic) if its not clear.